odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/mapper/RelationshipMapper.java

```diff
@@ -0,0 +1,97 @@
+package org.opendatadiscovery.oddplatform.mapper;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.opendatadiscovery.oddplatform.api.contract.model.DatasetERDRelationshipDetails;
+import org.opendatadiscovery.oddplatform.api.contract.model.DatasetGraphRelationshipDetails;
+import org.opendatadiscovery.oddplatform.api.contract.model.DatasetRelationship;
+import org.opendatadiscovery.oddplatform.api.contract.model.DatasetRelationshipList;
+import org.opendatadiscovery.oddplatform.api.contract.model.DatasetRelationshipType;
+import org.opendatadiscovery.oddplatform.api.contract.model.PageInfo;
+import org.opendatadiscovery.oddplatform.dto.RelationshipDetailsDto;
+import org.opendatadiscovery.oddplatform.dto.RelationshipDto;
+import org.opendatadiscovery.oddplatform.dto.RelationshipTypeDto;
+import org.opendatadiscovery.oddplatform.utils.Page;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class RelationshipMapper {
+    private final ErdRelationshipMapper erdRelationshipMapper;
+    private final GraphRelationshipMapper graphRelationshipMapper;
```

Comment:
I think we don't need that mapper to be dependant of ERD or Graph cases. For relationships page we need just `Relationship` object without details.  For details page FE will sequently request relationship and after that ERD or Graph details. So I think we should decompose this mapper